README.md

```diff
@@ -1,23 +1,12 @@
-<h5 align="center">
-  <br>
-  <a href="https://www.asyncapi.org"><img src="https://github.com/asyncapi/parser-nodejs/raw/master/assets/logo.png" alt="AsyncAPI logo" width="200"></a>
-  <br>
-  AsyncAPI CLI
-</h5>
-<p align="center">
-  <em>CLI to work with your AsyncAPI files. Currently supports validation, but it is under development for more features.</em>
-</p>
-
-<p align="center">
-<a href="https://github.com/asyncapi/cli/blob/master/LICENSE"><img alt="GitHub license" src="https://img.shields.io/github/license/asyncapi/cli"></a>
-<a href="https://github.com/asyncapi/cli/actions/workflows/if-nodejs-pr-testing.yml">
-<img src="https://github.com/asyncapi/cli/actions/workflows/if-nodejs-pr-testing.yml/badge.svg" alt="PR testing - if Node project"  />
-</a>
-<a href="https://www.npmjs.com/package/@asyncapi/cli">
-<img alt="npm" src="https://img.shields.io/npm/dw/@asyncapi/cli">
-</a>
-
-</p>
+[![AsyncAPI CLI](./assets/logo.png)](https://www.asyncapi.com)
+
+*CLI to work with your AsyncAPI files. Currently supports validation, but it is under development for more features.*
```

Comment:
Do we need italic formatting here?

Reply:
I would say probably not, other repos don't have the description in italics

Reply:
So @amanbedi1 please remove that formatting and rest is good :)